{
  setDebug(true);
  openTestURL();
  sleep(1000);
  assertFalse("Unexpected exception is visible.",$(NotificationElement.class).exists());
}
